src/lib/parser/index.ts

```diff
@@ -35,11 +35,16 @@ const findWord = (str: string): KeyWord | null => {
   return wordArr ?  wordArr[0] : null;
 };
 
+const replaceWord = (str: string): string => {
+    const regex = /[TODO:FIXME:]/g;
+    return str.replace(regex, "");
```

Comment:
chain with trim function to get rid of white spaces at end and start of sentences. http://devdocs.io/javascript/global_objects/string/trim